Phonebook.Frontend/src/app/shared/states/App.state.ts

```diff
@@ -19,6 +20,15 @@ export class SetSendFeedback {
   constructor(public sendFeedback: boolean) {}
 }
 
+export class SetTheme {
```

Comment:
are we missing here an interface for the theme classes? Something like

```typescript
interface ITheme{
   readonly type: string;
}
```

Reply:
nope that's just a class for performing the action. Has nothing to do with the Theme enum per se